servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -149,27 +152,28 @@ public String getServiceName() {
 
   @Override
   public void processJournalEntry(JournalEntry entry) throws IOException {
-    if (entry instanceof InodeEntry) {
-      mInodeTree.addInodeFromJournal((InodeEntry) entry);
-    } else if (entry instanceof InodeLastModificationTimeEntry) {
-      InodeLastModificationTimeEntry modTimeEntry = (InodeLastModificationTimeEntry) entry;
+    Message innerEntry = JournalProtoUtils.unwrap(entry);
```

Comment:
This comment is totally out of the scope of this PR, but might worth a discussion: 
instead of trying out each `if`, possible to register all handler according the to class name, and simply call the corresponding handler registered with the message type? In that way, we may able to speed up the process rate